NSS-Day-Cohort-66/rare-client-rare-poultry
# Pm post details comments integration

# Post Comments

## Created the link to a Post Comments Through Post Details
* Added a link in Post Details Component
* Added a route in application views
* Added useParams in Comments component
* Added error handling in Comments component

## Steps to test 
1. git fetch
2. git checkout pm-postDetails-comments-integration
3. 1pm run dev
4. click on a post
5. click on comments at bottom of container
6. double check with database to make sure all comments that should be there are there
